web-common/src/main/java/edu/unc/lib/boxc/web/common/utils/SerializationUtil.java

```diff
@@ -113,43 +113,43 @@ public static Map<String, Object> metadataToMap(ContentObjectRecord metadata, Ac
         }
 
         if (metadata.getId() != null) {
-            result.put("id", metadata.getId());
+            result.put(SearchFieldKey.ID.getUrlParam(), metadata.getId());
         }
 
         if (metadata.getTitle() != null) {
-            result.put("title", metadata.getTitle());
+            result.put(SearchFieldKey.TITLE.getUrlParam(), metadata.getTitle());
         }
 
         if (metadata.get_version_() != null) {
-            result.put("_version_", metadata.get_version_());
+            result.put(SearchFieldKey.VERSION.getUrlParam(), metadata.get_version_());
```

Comment:
these aren't actually equivalent at the moment, urlParam doesn't have the _'s. And while that is what we ultimately want, we'd have to update the UI code to expect the correct name, which might be out of scope for this ticket